### DISCUSSION TITLE
Documentation of the contents of `data` and `error` from auth JS code?

### DISCUSSION THREAD

**Discussion by davetron5000** [supabase/supabase#11515] [Questions]
Several of the auth methods liked `signUp` and `signInWithPassword` don't document their return values, or at least I can't find where they are documented.  Is there somewhere else that these might be documented? I realize the docs are generated from code somehow, so is this code open source somewhere and perhaps I can provide docs or have a more direct discussion there? I looked in supabase-js and postgrest-js briefly and couldn't see anything obvious.

**Comment by j4w8n** [comment]
I was thinking there was some small talk about documenting return types.

But to answer your question, the code is in the gotrue repos. Here's the js one:

https://github.com/supabase/gotrue-js/blob/master/src/GoTrueClient.ts
